packages/devtools_app/lib/src/screens/vm_developer/vm_object_model.dart

```diff
@@ -146,6 +148,51 @@ class FuncObject extends VmObject {
 
   @override
   SourceLocation? get _sourceLocation => obj.location;
+
+  late final FunctionKind? kind;
```

Comment:
For the properties that don't require an asynchronous request, can we just make them getters instead of copying the values from `obj`? For example, something like this:

```dart
bool? get isInlineable => obj,inlineable;
```

Reply:
Sure